conf/broker.conf

```diff
@@ -312,6 +312,9 @@ delayedDeliveryEnabled=true
 # Default is 1 second.
 delayedDeliveryTickTimeMillis=1000
 
+# Whether to enable acknowledge of batch local index.
+batchIndexAcknowledgeEnable = true
```

Comment:
```suggestion
batchIndexAcknowledgementEnabled = true
```

Reply:
a better name for this feature would be "acknowledgment at batch index level". So we can name this setting "acknowledgmentAtBatchIndexLevelEnabled".